Formula/apache-drill.rb

```diff
@@ -1,9 +1,8 @@
 class ApacheDrill < Formula
   desc "Schema-free SQL query engine for Hadoop and NoSQL"
   homepage "https://drill.apache.org/download/"
-  url "https://www.apache.org/dyn/closer.cgi?path=drill/drill-1.7.0/apache-drill-1.7.0.tar.gz"
-  mirror "https://archive.apache.org/dist/drill/drill-1.7.0/apache-drill-1.7.0.tar.gz"
```

Comment:
Looks like that was never a correct mirror in the first place (`closer.cgi` loads an HTML page).

@hrbrmstr Here's a potential new mirror: http://mirror.cc.columbia.edu/pub/software/apache/drill/drill-1.9.0/apache-drill-1.9.0.tar.gz